shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -932,7 +932,7 @@ - (void)testHideOverlay {
   engine.viewController = nil;
 }
 
-- (void)testHideA11yElements {
+- (void)testDoNotHideA11yElements {
```

Comment:
Maybe we should just rip out this test now that we are undoing the work that caused it.